### ISSUE TITLE
Add a feature only once

### ISSUE THREAD

**Issue by amuino** [jrallison/rollout_ui#17]
The code for `active?` in `monkey_patch.rb` always adds a feature to the Redis `features` set.

Is there any reason why this can't be done just once (I'm thinking on keeping a ruby Set or Hash with the features that have already been added).

Redis is fast, but doing nothing is still faster.

I could submit a pull request for this.

**Comment by kyleries** [comment]
Thanks for logging this issue (and your patience)! I'm tending the garden, so to speak, and will be closing this as a duplicate of #28. Feel free to weigh in on that PR and/or experiment with that solution.
